src/mpl2/src/SimulatedAnnealingCore.cpp

```diff
@@ -510,11 +542,6 @@ void SimulatedAnnealingCore<T>::fastSA()
   if (graphics_) {
     graphics_->startSA();
   }
-  // perturb();  // Perturb from beginning
-  std::iota(pos_seq_.begin(), pos_seq_.end(), 0);
-  std::iota(neg_seq_.begin(), neg_seq_.end(), 0);
-  std::iota(pre_pos_seq_.begin(), pre_pos_seq_.end(), 0);
-  std::iota(pre_neg_seq_.begin(), pre_neg_seq_.end(), 0);
```

Comment:
What has replaced this?

Reply:
this std::iota usage looks unneeded due to the way we initialize the sequences:

https://github.com/The-OpenROAD-Project/OpenROAD/blob/bdda927aed0ece8efa42706bd847024ce9ec5f37/src/mpl2/src/SimulatedAnnealingCore.cpp#L114-L123